chef_master/source/resource_openssl_ec_private_key.rst

```diff
@@ -3,7 +3,7 @@ openssl_ec_private_key
 =====================================================
 `[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/resource_openssl_ec_private_key>`__
 
-Use the **openssl_ec_private_key** resource to generate elliptic curve (EC) private key files. If a valid EC key file can be opened at the specified location, no new file will be created. If the EC key file cannot be opened or does not exist, it will be overwritten.
+Use the **openssl_ec_private_key** resource to generate an elliptic curve (EC) private key files. If a valid EC key file can be opened at the specified location, no new file will be created. If the EC key file cannot be opened or does not exist, it will be overwritten.
```

Comment:
Is the resource used to generate one  elliptic curve (EC) private key file or multiple? "Files" will need to be changed if the former, or "an" will need to be change if the latter.

Reply:
I fixed this and a few other issues.